yjit.rb

```diff
@@ -253,6 +253,11 @@ def _print_stats # :nodoc:
       # Number of failed compiler invocations
       compilation_failure = stats[:compilation_failure]
 
+      if stats[:x86_call_rel32] || stats[:x86_call_reg]
```

Comment:
I think the intention is to check for 0 here. If so, 0 is truthy in Ruby, so this should show on ARM builds?